src/home/HomeCookieSnackbar.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from 'react';
+import {Button, Snackbar, IconButton, useMediaQuery} from '@mui/material';
+import CloseIcon from '@mui/icons-material/Close';
+import Slide, { SlideProps } from '@mui/material/Slide';
+import theme from '../theme';
+
+
+type CookieSnackbarProps = 
+{
+    timeToWait: number;
+}
+
+type TransitionProps = Omit<SlideProps, 'direction'>;
+
+function TransitionUp(props: TransitionProps) {
+    return <Slide {...props} direction="up" />;
+}
+
+const HomeCookieSnackbar = ({timeToWait} : CookieSnackbarProps) =>
+{
+    const isDesktop = useMediaQuery(theme.breakpoints.down('lg'));
+    const [open, setOpen] = React.useState(false);
+    const [cookieMode, setCookieMode] = React.useState<boolean>();
+
+    const handleClick = (SelectedMode: boolean) => () => 
+    {
+        setOpen(false);
+        setCookieMode(SelectedMode);
+    }
+
+    const cookieAction = (
+        <React.Fragment>
+          <Button color="primary" size="small" onClick={handleClick(true)}>
```

Comment:
let's add some spacing between the two buttons. an extra 12px of right margin should be fine